Exceptions/UiPageIdNotPresentError.php

```diff
@@ -0,0 +1,17 @@
+<?php
+namespace exface\Core\Exceptions;
+
+/**
+ * Exception thrown if a UI page has no UID.
+ *
+ * @author SFL
+ *
+ */
+class UiPageIdNotPresentError extends RuntimeException
```

Comment:
Lieber UiPageIdMissingError. So wie jetzt stellt sich sofort die Frage "not present where?"